snapcraft.yaml

```diff
@@ -36,52 +36,45 @@ parts:
   libsane:
     source: https://salsa.debian.org/debian/sane-backends.git
     source-type: git
-    source-tag: upstream/1.0.27
+    source-tag: 'upstream/1.0.27'
     plugin: autotools
-    autotools-configure-parameters: [--prefix=/snap/simple-scan/current/usr, --with-api-spec=no]
-    organize:
-      snap/simple-scan/current/usr: usr
-    build-packages:
-      - autotools-dev
-      - autoconf
-      - chrpath
-      - dh-systemd
-      - libavahi-client-dev
-      - libgphoto2-dev
-      - libieee1284-3-dev
-      - libjpeg-dev
-      - libltdl3-dev
-      - libsystemd-dev
-      - libtiff-dev
-      - libusb-1.0-0-dev
-      - pkg-config
-      - po-debconf
-      - xutils-dev
+    autotools-configure-parameters:
+      - --prefix=/usr
+      - --with-api-spec=no
 
   simple-scan:
     after: [libsane]
     source: https://gitlab.gnome.org/GNOME/simple-scan.git
     source-type: git
-    source-tag: '40.6'
+    source-tag: '42.1'
     plugin: meson
-    meson-parameters: [--prefix=/usr]
-    organize:
-      snap/simple-scan/current/usr: usr
+    parse-info: [usr/share/metainfo/simple-scan.appdata.xml]
+    meson-parameters:
+      - --prefix=/usr
     build-packages:
       - desktop-file-utils
       - gnome-common
       - intltool
       - libcolord-dev
       - libgusb-dev
       - libpackagekit-glib2-dev
-      - libsane-dev
     override-pull: |
-      snapcraftctl pull
-      snapcraftctl set-version $(git describe --tags --abbrev=10)
-        
+      craftctl default
+      craftctl set version=$(git describe --tags --abbrev=10)
```

Comment:
Since the version exists [in the upstream appdata file](https://gitlab.gnome.org/GNOME/simple-scan/-/blob/master/data/simple-scan.appdata.xml.in#L37), I think we can remove this.

Reply:
You are right. Done. Will check all other snaps.